Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -436,7 +528,7 @@ function Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric
         if ($($ServiceFabricRemediated | Measure-Object).Count -gt 0)
         {
             Write-Host "Successfully set the ClusterProtectionLevel to 'EncryptAndSign' on the following ServiceFabric(s) in the subscription:" -ForegroundColor $([Constants]::MessageType.Update)
-           
```

Comment:
PS: Did you check that logs are being saved in log files because I am not able to see that block of code as well.

Reply:
hey Saurabh, I have verified, able to push the changes to log files